src/utils/checkAbilities.ts

```diff
@@ -15,5 +15,8 @@ export default function checkAbilities(role: String, eventType: String, componen
         case "user":
             response = userAbility.can(eventType, component);
             return response;
```

Comment:
Please add "consumer" case at checkAbilities.ts as well, 


```
        case "consumer":
            response = consumerAbility.can(eventType, component);
            return response;
```

Reply:
Thank you for the feedback, this is done